src/api/app/helpers/webui/webui_helper.rb

```diff
@@ -261,10 +261,10 @@ def user_and_role(user, role = nil, options = {})
       icon = user_icon(user)
     end
 
-    if !(real_name.empty? || opt[:short])
-      printed_name = "#{real_name} (#{user})"
-    else
+    if real_name.empty? || opt[:short]
```

Comment:
I am not the right one to answere this, since I did not write the code. But if you want I can remove that part from the commit.